dashboards-observability/public/components/application_analytics/home.tsx

```diff
@@ -308,7 +308,7 @@ export const Home = (props: HomeProps) => {
     };
 
     return http
-      .patch(`${APP_ANALYTICS_API_PREFIX}/`, {
+      .put(`${APP_ANALYTICS_API_PREFIX}/`, {
```

Comment:
does put work for update? i think other objects use post for updates

Reply:
It should. I think the convention is to use post for creating objects and put/patch is for updating??